### PR TITLE
update fee example

### PR DESCRIPTION
**Pull request by ubbabeck**
Update fee example to use new open_channel_fee function https://github.com/breez/breez-sdk/pull/416.

### CODE REVIEW

File: src/guide/receive_onchain.md [hunk 1]
```diff
@@ -450,190 +450,52 @@ To calculate the fees for a channel being opened by the LSP:
...
-const isChannelOpeningFeeNeeded = async (amountMsats: number): boolean => {
-    try {
-        const nodeInfo = await nodeInfo()
-        return nodeInfo.inboundLiquidityMsats <= amountMsats            
-    } catch (error) {
-        // handle error
-    }
-    return false
-}
+// TODO add example for openChannelFee
```
Comment (dangeross) [inline src/guide/receive_onchain.md:483]:
```
const amountMsat = <amount msat>
try {
    const channelFees = await openChannelFee({amountMsat: amountMsat})
} catch (error) {
    // handle error
}
```

File: src/guide/receive_onchain.md [hunk 2]
```diff
@@ -642,228 +504,36 @@ bool isChannelOpeningFeeNeeded(int amountMsat) async {
...
 ```go
-func isChannelOpeningFeeNeeded(amountMsats uint64) bool {
-	nodeInfo, err := sdkServices.NodeInfo()
-	if err != nil {
-        // Handle error
-	}
-	return nodeInfo.InboundLiquidityMsats <= amountMsats 
-}
+amountMsat := <amount msat>
+channelFees, err := sdkServices.OpenChannelFee(breez_sdk.OpenChannelFeeRequest(amountMsat))
```
Comment (dangeross) [inline src/guide/receive_onchain.md:522]: channelFees, err := sdkServices.OpenChannelFee(breez_sdk.OpenChannelFeeRequest{AmountMsat: amountMsat})

**Review by dangeross — CHANGES_REQUESTED** [review]

File: src/guide/receive_onchain.md (hunk 1 above)
Comment (ubbabeck) [reply src/guide/receive_onchain.md]: Thanks. Added

File: src/guide/receive_onchain.md [hunk 3]
```diff
@@ -450,178 +450,41 @@ To calculate the fees for a channel being opened by the LSP:
...
 
-```kotlin,ignore
-fun calculateChannelOpeningFee(amountMsats: Long): Long? {
-    val channelOpeningFeeNeeded = isChannelOpeningFeeNeeded(amountMsats)
-    if (channelOpeningFeeNeeded) {
-        return calculateFeesForAmount(amountMsats)
-    }
-    return null
-}
+```kotlin, ignore
```
Comment (ok300) [inline src/guide/receive_onchain.md:474]:
```suggestion
```kotlin,ignore
```

NIT: The syntax here is without the space

**Review by ok300 — APPROVED** [review]

**Pull request merged**